mk/generate.mk

```diff
@@ -78,7 +78,8 @@ generate/kumapolicy-gen/%: build/protoc-gen-kumapolicy generate/dirs/%
 		--kumapolicy_opt=endpoints-template=$(TOOLS_DIR)/policy-gen/templates/endpoints.yaml \
 		--kumapolicy_out=$(POLICIES_DIR)/$* \
 		--plugin=protoc-gen-kumapolicy=$(BUILD_ARTIFACTS_DIR)/protoc/protoc-gen-kumapolicy \
-		$(POLICIES_DIR)/$*/api/*/*.proto ;
+		$(POLICIES_DIR)/$*/api/*/*.proto ; \
+	sed -i.bak 's/,omitempty//g' $(POLICIES_DIR)/$*/api/*/*.go && rm $(POLICIES_DIR)/$*/api/*/*.bak
```

Comment:
I'm confused I thought we were not going to do this. Can you explain this to me a little more? I also believe you should be able to set sed to just not keep a backup

Reply:
This PR doesn't remove protos, so we have to use `sed` while we have protos. 
>I also believe you should be able to set sed to just not keep a backup

This backup is a hack for sed to work both on MacOS and Ubuntu. Otherwise different parameters are required:
```
# darwin
$ sed -i '' 's/,omitempty//g' $(POLICIES_DIR)/$*/api/*/*.go
# ubuntu
$ sed -i 's/,omitempty//g' $(POLICIES_DIR)/$*/api/*/*.go
```
I found the hack here https://unix.stackexchange.com/a/381201

Reply:
:(